h2o-r/h2o-package/R/automl.R

```diff
@@ -1,15 +1,9 @@
 #' Automatic Machine Learning
-#'
+#' 
 #' The Automatic Machine Learning (AutoML) function automates the supervised machine learning model training process.
-#' The current version of AutoML trains and cross-validates the following algorithms:
-#' three pre-specified XGBoost GBM (Gradient Boosting Machine) models, a fixed grid of GLMs,
-#' a default Random Forest (DRF), five pre-specified H2O GBMs, a near-default Deep Neural Net,
-#' an Extremely Randomized Forest (XRT), a random grid of XGBoost GBMs, a random grid of H2O GBMs,
-#' and a random grid of Deep Neural Nets. In some cases, there will not be enough time to complete all the algorithms,
-#' so some may be missing from the leaderboard. AutoML trains several Stacked Ensemble models during the run.
-#' Two kinds of Stacked Ensemble models are trained one of all available models, and one of only the best models of each kind.
-#' Note that Stacked Ensemble models are trained only if there isn't another stacked ensemble with the same base models.
-#'
```

Comment:
@ledell same as for Python description.
Given the R constraints in this main description (stops after the first blank line apparently), I moved the general legacy `@details` section to the top, and put the internals description into the new `@details` section, as this way I can use markdown there.